lib/cdo/honeybadger.rb

```diff
@@ -0,0 +1,61 @@
+require 'honeybadger'
+require_relative 'env'
+
+# Honeybadger extensions for command error logging
+module Honeybadger
+  class << self
+
+  # notify_command_error - log an error from an executed command to honeybadger.
```

Comment:
I think it would be clearer to indent class << self. Or maybe even just def self.etc.. I don't think we customarily use class << self